Update metrics-definitions.md

enelson720 opened the pull request:
Fixing MAU syntax  and adding additional information in the metric definition file.

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

comharris commented on source/leadership/leadership-meetings.md:
```diff
@@ -207,7 +218,7 @@ We're currently focused on quarterly plans and will move to halves
-1. CEO discusses company and department V2MOMs 1-1 with department heads to ensure alignment 
+1. CEO discusses company and department VPMOMs 1-1 with department heads to ensure alignment 
    -  If applicable during the period, sales, CEO and finance set or adjust the revenue targets  
 2. People meets with departments head to discuss current and future org structure  
```
should this be "department heads"?

jasonblais replied:
@comharris Thanks, updated!

jasonblais commented:
Changes reflected at https://github.com/mattermost/mattermost-handbook/pull/85

enelson720 committed:
Update metrics-definitions.md

Fixing MAU syntax  and adding additional information in the metric definition file.

jasonblais commented:
@rbradleyhaas for your review. You also have permissions to merge pull requests that change the BizOps content like definitions: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook#approved-reviewers-and-permissions

rbradleyhaas commented:
LGTM :shipit:
